style.css

```diff
@@ -0,0 +1,383 @@
+  font-size: 1rem;
+}
+
+.program {
+  background-image: url(./imgs/download.svg);
+  background-color: rgba(0, 0, 0, 0.918);
+  padding: 2rem 0.6rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.program h2,
+.speakers h2,
+.partner h2 {
+  color: #fff;
+  text-align: center;
+  font-size: 20px;
+  font-weight: 500;
+  position: relative;
+  margin-bottom: 1rem;
+}
+
+.program h2::before,
+.speakers h2::before,
+.partner h2::before {
+  content: '';
+  position: absolute;
+  left: 36%;
+  bottom: -50%;
+  width: 50px;
+  height: 3px;
+  background-color: rgba(204, 41, 41, 0.596);
+}
+
+.program button {
+  width: 80%;
+  height: 4rem;
+  font-size: 16px;
+  border: none;
+  background-color: rgba(204, 41, 41, 0.774);
+  color: #fff;
+}
+
+.speakers button {
+  border: 1px solid rgb(170, 170, 170);
+  width: 90%;
+  height: 2.5rem;
+  background-color: #fff;
+  justify-self: center;
+  cursor: pointer;
+}
+
+.program button:hover {
+  background-color: rgba(204, 41, 41, 0.863);
+  color: rgb(255, 255, 255);
+}
+
+.program .pros {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: rgba(255, 255, 255, 0.199);
+  color: #fff;
+  font-size: 1rem;
+  width: 100%;
+  margin: 2rem 0;
+  padding: 0.3rem 0.1rem;
+}
+
+.pros h4 {
+  color: #ec5242;
+  font-size: 1.178rem;
+  width: 30%;
+  margin-left: 1rem;
+}
+
+.pros i {
+  font-size: 2.5rem;
+  color: #d3d3d3;
+  margin-left: 0.5rem;
+}
+
+.speakers {
+  padding: 2rem 0;
+  background-color: #d3d3d3;
+  font-size: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.speakers h2 {
+  color: #000;
+  margin-top: 1rem;
+  margin-bottom: 5rem;
+}
+
+.speakers h2::before {
+  left: 42%;
+}
+
+.speaker-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: flex-start;
+}
+
+.speakers .speaker {
+  display: flex;
+  justify-content: start;
+  align-items: center;
+}
+
+.speaker img {
+  width: 180px;
+  height: 180px;
+  background-image: url(./imgs/chees.png);
+  background-position: 17% 17%;
+  background-repeat: no-repeat;
+  background-size: 70px 70px;
+  padding: 2rem;
+}
+
+.speaker-id {
+  margin-left: 0;
+  margin-top: -0.4rem;
+}
+
+.speaker-id h3 {
+  margin-bottom: 0.5rem;
+}
+
+.speaker #job {
+  color: #ec5242;
+  margin-bottom: 0.6rem;
+  font-size: 14px;
+}
+
+.speakers button i {
+  color: #ec5242;
+  margin-left: 0.5rem;
+}
+
+.partner h2 {
+  color: rgba(255, 255, 255, 0.808);
+  margin: 1rem 0 1rem 0;
+}
+
+.partner h2::before {
+  left: 43.5%;
+}
+
+.partner {
+  padding: 1rem 0;
+  background-color: #272a31;
+  padding-bottom: 2rem;
+}
+
+.partner-list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 1rem;
+  margin-top: 4rem;
+}
+
+.partner-list img {
+  width: 90px;
+  cursor: pointer;
+}
+
+footer {
+  background-color: #d3d3d3;
+  padding: 1rem;
+  font-size: 70%;
```

Comment:
- [ ] Great job 👏 👏. Seriously, your design looks great 💯. A small adjustment I would suggest you make is about the background color of the Footer section of the About page. It looks very dark on the Design while yours is light grey. 
It is important to pay attention to all details even the small ones because it is what makes you outstanding 😉
**Here is how the design looks 👇** 
![image](https://user-images.githubusercontent.com/90222110/181580468-429f88eb-f8f9-4f90-8332-8b849c6226a4.png)

**Here is your work 👇** 
![image](https://user-images.githubusercontent.com/90222110/181580913-1a3604b2-ab01-4666-a282-0d9b38d7a82d.png)

Please adjust the background color to match the one on the design 😃 

**NOTE** I realized that you have added the `our partners`  section above the footer section in the about section. I think it's fine too **but I think it would be better if you gave it a background color which will visually differentiate it from the footer as the footer is the one to have the dark background color**